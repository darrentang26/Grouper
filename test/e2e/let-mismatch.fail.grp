let Int x = "hi"
    in print x 