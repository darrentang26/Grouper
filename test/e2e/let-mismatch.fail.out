Fatal error: exception Semant.Failure("the left- and right-hand sides of bindings must mach: Int =/= String")
